### PR TITLE
Terrapin parameters

### PR DESCRIPTION
**Pull request by mortenfyhn**

### CODE REVIEW

File: vortex/config/robots/terrapin.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# When looking at the ROV from above, the thruster placement is:
+#
+#    _______
+# 8 /       \ 1
+#  |         |      N
+# 7|         |2     ^
+#  |         |   W<-|->E
+# 6|         |3     v
+#  |         |      S
+# 5 \_______/ 5
```
Comment (sindrehan) [inline vortex/config/robots/terrapin.yaml]: Should one of these be thruster nr 4?

**Review by sindrehan — CHANGES_REQUESTED** [review]
What is the plan for adding the actual parameters and not just dummy values? Should we delay the merging of this branch until we have them?

File: vortex/config/robots/terrapin.yaml (hunk 1 above)
Comment (mortenfyhn) [reply vortex/config/robots/terrapin.yaml]: Yes.

**Comment by mortenfyhn** [comment]
I reckon it's nice to have dummy parameters that let us run the system, on master. We can keep the branch alive or make a new branch when we have real values. Or maybe just commit those on master.

**Review by sindrehan — APPROVED** [review]

**Pull request merged**